resources/views/mail/site_invitation.blade.php

```
@@ -0,0 +1,13 @@
+@component('mail::message')
+Hello{{ $invitation->recipient->name ? ' ' . $invitation->recipient->name : '' }},
```

Is there a chance a recipient is not present? I'd rather throw an exception and fail if a recipient does not exist.

---

This can be checked in the Mail class.

---

I don't think it's possible for a recipient to not be present. Tested by attempting to create a SiteInvitation with no recipient (error: no default value), attempting to update an existing site invitation by setting the recipient to null (error: cannot be null), attempting to update an existing site invitation by setting the recipient to a non-existent id (error: foreign key constraint fails), and by sending an invitation, then deleting the user it was sent to (invitation is deleted when the recipient is deleted).